<!-- Resume template loosely from https://mszep.github.io/pandoc_resume/ -->

# &middot; [Aldrin Montana](http://research.aldrinmontana.com)

[Email](mailto:mail@aldrinmontana.com)

[LinkedIn](https://linkedin.com/in/drinmontana)

[GitHub](https://github.com/drin)

[DockerHub](https://hub.docker.com/u/octalene)

[OrcID](https://orcid.org/0000-0003-2073-4813)

[dblp](https://dblp.org/pers/hd/m/Montana:Aldrin)

---

Education
---------

2018-202X (does anyone really know)
:   **Ph.D., Computer Science**; UC, Santa Cruz

    Declarative Programmable Storage
    
    Advisors: Peter Alvaro, Josh Stuart

2007-2013
:   **B.S.-M.S., Computer Science**; Cal Poly, San Luis Obispo

    Clustering for Microbial Source Tracking
    
    Advisor: Alex Dekhtyar

Work Experience
---------------

June 2022 - Dec 2022
:   **Voltron Data (Remote)**; Software Engineer Intern, Data Engine

May 2020 - June 2021
:   **Seagate (Fremont, CA)**; Software Engineer Intern


April 2015 - September 2018
:   **Personalis (Menlo Park, CA)**; Bioinformatics Analyst/Engineer

    Content, Pipeline, and Annotation Team

    Software development using primarily bash, python, SQL server, and MySQL.

    * Process and load annotation data into internal database systems. Types
      of annotation data include:
        * Reference assemblies
        * Allele and genotype frequencies
        * Gene to phenotype relationships
        * Small variant to phenotype relationships

    * Maintain internal tools for annotating small variants called by
      internal variant calling pipeline. Internal tools leverage 3rd party
      tools, such as snpEff.

    * Contribute to data management including:
        * data model design
        * stored procedure and query development
        * system architecture
        * develop data reporting system


August 2013 - April 2015
:   **Workday (Pleasanton, CA)**; Associate Software Development Engineer

    OMS Performance - Scalability Engineering
    Software development using primarily java, bash, MySQL, and hive.

    * Handled occasional performance regressions in operations.
    * Contributed to testing platform for performance and regression tests.
    * Performed some load testing of internal high-volume test tenants using
      LoadRunner.
    * Performed some data analysis of application usage data to help
      understand and triage performance issues.

Technical Experience
--------------------

: **Experienced Programming Languages**
    
    Python, Java, Bash, Perl, C

: **Acquainted Programming Languages**

    Scala, Rust, Javascript

Volunteering
------------

: FASTER-STEAM (October 2017 - Present)

    Tech Committee, Mentor

: OutOfOfficeHours (June 2016 - Present)

    Mentor

: Code2040 (June 2016 - September 2016)

    Mentor

: Dublin HS Engineering & Design Academy (August 2014 - June 2016)

    Mentor

Publications
------------

* Jennifer Yen, et al., [A variant by any name][variant-annotation]

* Elo Leung, et al., [Protein Sequence Annotation Tool (PSAT)][psat]

* Aldrin Montana, et al., [Ontological Hierarchical Clustering for Library-Based Microbial Source Tracking][ohclust-2013]

* Alex Dekhtyar, et al., [Teaching Bioinformatics in Concert][teach-bio]

* Emily Neal, et al., Integrating Molecular Forensics into the Undergraduate
Science, Technology, Engineering and Mathematics Curriculum

* Anya Goodman, et al., [Pyroprinting: novel pyrosequencing-based method for studying E. coli
  diversity and microbial source tracking][pyroprinting]

* Aldrin Montana, [Chronology-Sensitive Hierarchical Clustering of Pyrosequenced DNA Samples of E. coli: A Case Study][ohclust-case-study]

Posters
-------

* Jan Lorenz Soliman, et al., Microbial Source Tracking by Molecular Fingerprinting (poster)

* Douglas Brandt, et al., Pyroprinting Sensitivity Analysis on the GPU (poster)


<!-- ----------------------------- -->
<!-- web links -->

<!-- volunteering -->
[faster-steam]: faster-steam.org
[ooohours]: outofofficehours.com
[code2040]: code2040.org
[dhs]: https://www.dublin.k12.ca.us/Page/7141

<!-- publications -->
[variant-annotation]: https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0396-7
[psat]: https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0887-y
[ohclust-2013]: https://ieeexplore.ieee.org/document/6753971
[teach-bio]: https://www.asee.org/documents/conferences/annual/2014/ASEE2014-BestPaper-Zone4.pdf
[pyroprinting]: https://www.fasebj.org/doi/abs/10.1096/fasebj.28.1_supplement.779.8
[ohclust-case-study]: https://ieeexplore.ieee.org/document/6120428
